### ISSUE TITLE
URLs are constant

### ISSUE THREAD

**Issue by ggtylerr** [5elenay/revoltgo#10]
The WebSocket and API URLs are constants, which means this package can't be used for self-hosted instances.

*(Sorry if I'm wrong on this, I don't know Go and haven't looked through the whole docs)*

**Comment by ben-forster** [comment]
You're correct, this package can't be used for self-hosted instances as it stands.

**Comment by prologic** [comment]
@ben-forster Do you plan on fixing this? Revolt.chat _can_ be self-hosted and I'm running one now. Was a bit surprised to find this library (_which looked nice_) hard-code the URL(s) :/
